### PR TITLE
#30: Display unique autocomplete suggestions

### PR DESCRIPTION
**Pull request by YuriyYatsyk**

### CODE REVIEW

File: src/test/java/com/appian/intellij/k/CompletionTest.java [hunk 1]
```diff
@@ -17,9 +17,9 @@ protected String getTestDataPath() {
   }
 
   public void testGlobals() {
-    testCompletion(new String[] {"globals1.k", "globals2.k"}, "square", "sum",
+    testCompletion(new String[] {"globals1.k", "globals2.k"},
         "someFnInDefaultNsInAnotherFile", "save", "scan", "scov", "sdev", "select", "set", "setenv", "show",
-        "signum", "sin", "sqrt", "ss", "ssr", "string", "sublist", "sum", "sums", "sv", "svar", "system");
+        "signum", "sin", "sqrt", "square", "ss", "ssr", "string", "sublist", "sum", "sums", "sv", "svar", "system");
```
Comment (YuriyYatsyk) [inline src/test/java/com/appian/intellij/k/CompletionTest.java:22]: "sum" function appeared twice in the expected results of the unit test, thus I did not need to create a new test, simply removed duplicate. I am also using computeIfAbsent() instead of putIfAbsent() to save on object creation, this resulted in slight order change - "square" now appears after "sqrt", which, imo, is where it should be anyway.

**Comment by a2ndrade** [comment]
@YuriyYatsyk thanks for the fix/contribution! I'll address #31 and #28 and ship a new version with all 3 issues fixed.
